sos/report/plugins/containers_common.py

```diff
@@ -17,11 +18,33 @@ class ContainersCommon(Plugin, RedHatPlugin, UbuntuPlugin):
     plugin_name = 'containers_common'
     profiles = ('container', )
     packages = ('containers-common', )
+    option_list = [
+        ('rootlessusers', 'colon-separated list of users\' containers info',
+         '', ''),
+    ]
```

Comment:
I think we need to decide on a standardized way to handle lists in plugin options like this. Other recent plugins use a space delimiter, is that possible here?

Reply:
Space as a separator isnt a problem for me. Just the syntax needs extra quotation marks to denote begin and end of the option value, like:

    -k containers_common.rootlessusers='jsmith bjones'

Colon is used in some plugin as well and does not need the quotes. But either works fine for me.

Sadly, the ideal separator - comma - hits an issue in how we parse the arguments (there is some older open issue on this, /me hit a blocker there, afaik).

Reply:
Followup from IRC: colons good, spaces not as good. Let's try to standardize on `:` delimiters in plugin options going forward.